src/Features/CSharp/Portable/UseLocalFunction/CSharpUseLocalFunctionDiagnosticAnalyzer.cs

```diff
@@ -197,10 +203,12 @@ private void SyntaxNodeAction(SyntaxNodeAnalysisContext syntaxContext, INamedTyp
 
         private bool CanReplaceAnonymousWithLocalFunction(
             SemanticModel semanticModel, INamedTypeSymbol expressionTypeOpt, ISymbol local, BlockSyntax block,
-            AnonymousFunctionExpressionSyntax anonymousFunction, CancellationToken cancellationToken)
+            AnonymousFunctionExpressionSyntax anonymousFunction, out ImmutableArray<Location> locations, CancellationToken cancellationToken)
         {
             // Check all the references to the anonymous function and disallow the conversion if
             // they're used in certain ways.
+            List<Location> delegateInvokeMethodLocations = null;
```

Comment:
just do: 

```c#
var explicitInvokeCalls = ArrayBuilder<Location>.GetInstance()```
...
explicitInvokeCalls.Add(location);
...
locations = explicitInvokeCalls.ToImmutableAndFree();
```

Reply:
Don't call this locations.  Call this 'explicitInvokeCallLocations'.